doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -185,6 +196,93 @@
     "plt.colorbar(label='RMSD (Angstrom)')"
```

Comment:
A thought, what do we thing about switching to the Å symbol instead of the written unit name? Just thinking asthetics wise if we want to follow a single "style" it'd be a good one to make a decision on.

Reply:
I like "Å" a lot more.